Report/sections/1_analysis/Importance_of_programming.tex

```diff
@@ -0,0 +1,56 @@
+\chapter{Importance of Programming} \label{chap:importance_of_programming}
+
+Humans have entered a digital age, as more and more stuff gets 
+digitalized every year. This transition has been ongoing for a few 
+generations, and the need for software cannot be understated. The 
+world runs on software, your coffee machine, television, car, mobile
+phone, and the list goes on, not to mention that every single business 
+and organization in the world needs an app, website, etc. Most companies
+need custom made applications or websites, and therefore a lot of coders
+is needed.
+
+This digital transition has made humans more dependant on technology than ever - 
+and for a good reason. The year 2020 when the pandemic hit the world is a good example of that.
+The pandemic changed the every day lives of a lot of people. Due to restrictions and curfew,
+people from all over the world had to stay home. This also applied to people 
+who were studying or working as well. Business still had work that needed to be done,
+students still had education they needed to finish. However, the world could not simply 
+standby and wait for the pandemic to go over. 
+
+In order to solve this problem,
+various digital solutions have been used to prevent the world economy from receding.
+Technology in form of video communication platforms such as Zoom and Microsoft Teams,
+have allowed students to be able to stll receive teaching/lectures and study material online. 
+Workers, who's work often consists of meetings and communication with other people,
+have also been able to continue doing there work from home through video communication platforms. 
+Restaurants and other businesses in the food industry have been able keep selling food, 
+even with restrictions prohibiting customers to enjoy their food at the restaurant, 
+they have shifted their sales to take away, where customers for example can order
+food from their website or through an app such as Wolt. \newline
```

Comment:
```suggestion
food from their website or through an app.
```
Use double linebreak instead of `\newline` - Does not break formatting.
Needs source